src/tools/mesh_tester.cpp

```diff
@@ -127,53 +130,65 @@ const std::set<RuntimePolicy> Input::s_validPolicies({
 
 void Input::parse(int argc, char** argv, CLI::App& app)
 {
-  app.add_option("-r,--resolution", resolution,
-                 "Resolution of uniform grid. \n"
-                 "Set to 1 to run the naive algorithm (without a spatial index). \n"
-                 "Set to less than 1 to use the spatial index with a resolution \n"
-                 "of the cube root of the number of triangles.")
-  ->capture_default_str();
-
-  app.add_option("-p, --policy", policy,
-                 "With \'-r 1\', set runtime policy. \n"
-                 "Set to 0 to use the sequential algorithm (w/o RAJA). \n"
-  #ifdef AXOM_USE_RAJA
-                 "Set to 1 to use the RAJA sequential policy. \n"
-    #ifdef AXOM_USE_OPENMP
-                 "Set to 2 to use the RAJA OpenMP policy. \n"
-    #endif
-    #ifdef AXOM_USE_CUDA
-                 "Set to 3 to use the RAJA CUDA policy."
-    #endif
+  app
+    .add_option(
+      "-r,--resolution",
+      resolution,
+      "Resolution of uniform grid. \n"
+      "Set to 1 to run the naive algorithm (without a spatial index). \n"
+      "Set to less than 1 to use the spatial index with a resolution \n"
+      "of the cube root of the number of triangles.")
+    ->capture_default_str();
+
+  app
+    .add_option("-p, --policy",
+                policy,
+                "With \'-r 1\', set runtime policy. \n"
+                "Set to 0 to use the sequential algorithm (w/o RAJA). \n"
+#ifdef AXOM_USE_RAJA
+                "Set to 1 to use the RAJA sequential policy. \n"
+  #ifdef AXOM_USE_OPENMP
+                "Set to 2 to use the RAJA OpenMP policy. \n"
+  #endif
+  #ifdef AXOM_USE_CUDA
+                "Set to 3 to use the RAJA CUDA policy."
   #endif
-  )
-  ->capture_default_str()
-  ->check(CLI::IsMember{Input::s_validPolicies});
+#endif
+                )
+    ->capture_default_str()
+    ->check(CLI::IsMember {Input::s_validPolicies});
 
-  app.add_option("-i,--infile", stlInput,"The STL input file")
-  ->required()
-  ->check(CLI::ExistingFile);
+  app.add_option("-i,--infile", stlInput, "The STL input file")
+    ->required()
+    ->check(CLI::ExistingFile);
```

Comment:
I think clang-format does a much better job with these chained function calls.
(Notice the improved indentation).